Dockerfile

```diff
@@ -1,10 +1,30 @@
+# Container image that runs your code
+FROM alpine:3.10
```

Comment:
I don't think we want to have a multi stage build of this image, and we don't want to base the image on alpine either - we have a requirement to base all of our images on `ubi`.